### ISSUE TITLE
你好，cols = nn.functional.unfold(up_x.permute(0, 2, 3, 1), self.kernel_size, padding=1)，该咋改呀？

### ISSUE THREAD

**Issue by NewboySoCool** [XuecaiHu/Meta-SR-Pytorch#81]

**Comment by NewboySoCool** [comment]
> 没改出来
你好，我找到3d unfold的办法，同时修改了成了3d meta_upsampler， 有时间的话可以帮忙看看是否符合2d设计。
[meta-upsampler3d.txt](https://github.com/XuecaiHu/Meta-SR-Pytorch/files/11770254/meta-upsampler3d.txt)
